src/ProjectTemplates/Web.Spa.ProjectTemplates/content/Angular-CSharp/ClientApp/aspnetcore-https.js

```diff
@@ -0,0 +1,40 @@
+// This script sets up HTTPS for the application using the ASP.NET Core HTTPS certificate
+const fs = require('fs');
+const spawn = require('child_process').spawn;
+const path = require('path');
+
+const baseFolder =
+  process.env.APPDATA !== undefined && process.env.APPDATA !== ''
+    ? `${process.env.APPDATA}/ASP.NET/https`
+    : `${process.env.HOME}/.aspnet/https`;
+
+const certificateName = process.argv.map(arg => arg.match(/--name=(?<value>.+)/i))
+                                    .filter(Boolean)
+                                    .reduce((previous, current) => previous || current.groups.value, undefined) ||
+  process.env.npm_package_name;
+
+if(!certificateName){
+  console.error('Invalid certificate name. Run this script in the context of an npm/yarn script or pass --name=<<app>> explicitly.')
+  process.exit(-1);
+}
+
+const certFilePath = path.join(baseFolder, `${certificateName}.pem`);
+const keyFilePath = path.join(baseFolder, `${certificateName}.key`);
+
+if (!fs.existsSync(certFilePath) || !fs.existsSync(keyFilePath)) {
+  spawn(
+    'dotnet',
+    [
+      'dev-certs',
+      'https',
+      '--export-path',
+      certFilePath,
+      '--format',
+      'Pem',
+      '--no-password',
+    ],
+    {
+      stdio: 'inherit',
+    }
+  ).on('exit', (code) => process.exit(code));
```

Comment:
By all means please ignore this suggestion if you don't like it, but the code might seem less intimidating and easier to skim-read if formatted closer to the JS-typical brace style and more compactly:

```suggestion
  spawn('dotnet', [
    'dev-certs', 'https',
    '--no-password',
    '--export-path', certFilePath,
    '--format', 'Pem',
  ], { stdio: 'inherit' })
  .on('exit', code => process.exit(code));
```

Reply:
vs code formatting is fighting me :). I'll fix it, it used to be that way